test/Kestrel.Transport.Sockets.FunctionalTests/Kestrel.Transport.Sockets.FunctionalTests.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <AssemblyName>Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.FunctionalTests</AssemblyName>
-    <RootNamespace>Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.FunctionalTests</RootNamespace>
+    <AssemblyName>Sockets.FunctionalTests</AssemblyName>
```

Comment:
No we know if the messes with TC test tracking?

Reply:
So @ryanbrandenburg and I are not completely sure about how test history are tracked but it seems to be the full name of the method which isn't affected by the assembly name. However, Ryan was okay with the test history being lost.

Reply:
Why change just the assembly name and not the project or directory name?

Reply:
Well *I'm* not OK with the history being lost if there's any reasonable alternative.

Reply:
@Tratcher only the assembly name is used to construct the log directory at {root}/artifacts/logs/... the logs are not stored in the test directory. @halter73 We believe that with this change, the history will not be lost, but we are not 100% sure.

Reply:
Shouldn't you be able to test this with a feature branch?

Reply:
Ah didn't know feature branches ran tests. For some reason I thought they only ran build.

Reply:
Yup confirmed that test history is maintained